spacy/lang/punctuation.py

```diff
@@ -24,7 +24,7 @@
         r"(?<=°[FfCcKk])\.",
         r"(?<=[0-9])(?:{c})".format(c=CURRENCY),
         r"(?<=[0-9])(?:{u})".format(u=UNITS),
-        r"(?<=[0-9{al}{e}{p}(?:{q})])\.".format(
+        r"(?<=[0-9{al}{e}{p}{q}])\.".format(
```

Comment:
This non-capturing group didn't seem to serve any useful purpose, so I removed it and confirmed that doing so didn't seem to have any impact on the tests. I'd be grateful for confirmation that there doesn't seem to be any reason for keeping it.

Reply:
Has this been included in a separate PR? I saw two but those were closed again? Just asking out of curiosity, if there was a reason not to go forward with this edit.

Reply:
I discovered that the groups did serve a purpose in the `CURRENCY` and `UNITS` regexes and thought perhaps there was an argument for maintaining the same structure throughout the regexes in the group, so that it was better to leave this as it is.